about.html

```diff
@@ -0,0 +1,171 @@
+          <h1 class="desktop-main-title">GHANA MICRONAUTS<br />MEETUP 2022</h1>
+          <h2 class="hero-sub-title">The Third Edition!</h2>
+        </div>
+        <div class="hero-paragraph about-hero-paragraph">
+          It's going to be a day of interaction with some of the world's most
+          promising software developers. You wouldn't want to miss this
+          opportunity to learn from seasoned devs. Connect and network with
+          other amazing software developers.
+        </div>
+        <p id="get-in-touch">
+          Please contact us via email for any further questions about the Ghana
+          Micronauts Meetup 2022.
+        </p>
+        <p class="our-email">info@ghanamicronauts.com</p>
+      </div>
+    </header>
+
+    <section class="more-info">
+      <h3 id="more-information-heading">More info</h3>
+      <img id="ghmmLogo-about" src="./images/ghmm-logo.png" alt="" />
+      <p class="program-brief">
+        The Ghana Micronauts Meetup is an initiative of Microverse students
+        risiding in Ghana. The next edition is going to be amazing You wouldn't
+        want to miss it. Mark it on your calendar.
+      </p>
+      <img
+        id="ghmm-photo"
+        src="./images/micronautsMeetup.jpg"
+        alt="Two Micronauts alumni"
+      />
+    </section>
+
+    <section class="previous-edition">
+      <h3 class="previous-edition-title">Highlights of Previous Editions</h3>
+      <div id="previous-editions-underline"></div>
+      <p>
+        Take a closer look at the first two meetups which took place at Takoradi
+        and Kumasi
+      </p>
+      <div class="previous-meetups">
+        <div class="previous-meetup-details">
+          <div class="first-edition-brief takoradiImage">
+            <h4>FIRST EDITION</h4>
+            <p>
+              Maiden edition of Ghana Micronauts Meetup held in Takoradi, Ghana.
+            </p>
+          </div>
+        </div>
+        <div class="previous-meetup-details">
+          <div class="second-edition-brief kumasiImage">
+            <h4>SECOND EDITION</h4>
+            <p>
+              Second edition of Ghana Micronauts Meetup held in Kumasi, Ghana.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="sponsors">
+      <div class="sponsors-heading">
+        <h5 class="sponsors-title">Sponsors / Partners</h5>
+        <div id="sponsors-underline"></div>
+      </div>
+      <div class="sponsors-logos">
+        <img src="./images/partners/bvmstudio-logo.png" alt="BVM Studio Logo" />
+        <img src="./images/partners/google-logo.png" alt="Google Logo" />
+        <img src="./images/partners/nita-logo.png" alt="Nita Logo" />
+        <img
+          src="./images/partners/microverse-logo.png"
+          alt="Microverse Logo"
+        />
+        <img src="./images/partners/github-logo.png" alt="GitHub Logo" />
+      </div>
+    </section>
```

Comment:
- [ ] I would suggest updating your design because according to the template, this sponsors section it's not shown on the about page for the desktop version, but should be on the mobile version. Please, kindly remove this section in order to match the original design :raised_hands:

| Mobile version | Desktop version |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/146961349-6aeaa2bd-41ef-45d4-a4be-447fa82da321.png) | ![image](https://user-images.githubusercontent.com/81522435/146960714-0355cc4d-5109-4ba6-b6fc-e1d0fa8e0569.png) |